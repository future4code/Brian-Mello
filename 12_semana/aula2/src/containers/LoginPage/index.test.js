import React from 'react'
import {shallow} from 'enzyme'
import renderer from "react-test-renderer";
import { LoginPage } from './index'

describe('Teste Container Do LoginPage', () => {

    it('Snapshot', () => {
        const loginPage = renderer.create(<LoginPage/>).toJSON();
        expect(loginPage).toMatchSnapshot();
    })
})