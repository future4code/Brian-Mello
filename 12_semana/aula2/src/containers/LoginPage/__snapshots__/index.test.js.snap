// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste Container Do LoginPage Snapshot 1`] = `
<div>
  <header
    className="sc-AykKD jAea-Do"
  >
    <img
      className="sc-fzXfNd dbbLyM"
      src="4eddit.png"
    />
  </header>
  <form
    className="sc-fzXfNi bnexOn"
    onSubmit={[Function]}
  >
    <h1>
      Log in
    </h1>
    <div
      className="MuiFormControl-root MuiTextField-root sc-fzXfMD kNKgYo"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Mui-required Mui-required"
        data-shrink={false}
      >
        E-mail
        <span
          className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
        >
           
          *
        </span>
      </label>
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiInputBase-input MuiInput-input"
          disabled={false}
          name="email"
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={true}
          type="email"
          value=""
        />
      </div>
    </div>
    <div
      className="MuiFormControl-root MuiTextField-root sc-fzXfMD kNKgYo"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Mui-required Mui-required"
        data-shrink={false}
      >
        Password
        <span
          className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
        >
           
          *
        </span>
      </label>
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiInputBase-input MuiInput-input"
          disabled={false}
          name="password"
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={true}
          type="password"
          value=""
        />
      </div>
    </div>
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text sc-fzXfMz jpfkqO"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="submit"
    >
      <span
        className="MuiButton-label"
      >
        Entrar
      </span>
    </button>
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text sc-fzXfMz jpfkqO"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Cadastro
      </span>
    </button>
  </form>
</div>
`;
